cmd/validator/validator.go

```diff
@@ -84,7 +86,7 @@ func validateFileTypeList(input []string) bool {
 		if len(t) == 0 {
 			continue
 		}
-		if !slices.Contains(types, t) {
+		if !slices.Contains(types, strings.ToLower(t)) {
```

Comment:
You will have to make sure the method that add the types convert to lowercase before adding to the slice types